client/src/pages/Campaign.js

```diff
@@ -81,6 +93,7 @@ const Campaign = (props) => {
       const cloudIcon = <CloudIcon className="fas fa-cloud" style={{color: "grey"}} />
 
       result.Prospects.map(prospect => {

```

Comment:
``result.prospects.map`` prospects should be lowercase, same with all the other keys, pls make that change in CampaignResource GET response!

Reply:
ok got it!